de_templates/ingest/Ingest_SourceType_SourceName/de-ingest-ado-pipeline.yml.jinja

```diff
@@ -76,6 +76,12 @@ stages:
           - template: ../../../build/azDevOps/templates/air-infrastructure-data-setup.yml
             parameters:
               TaskctlVersion: {% raw %}${{ variables.TaskctlVersion }}{% endraw %}
+          - script: pip install --upgrade pre-commit tomli
```

Comment:
do you think we should have a taskctl task instead that installs pre-commit and also runs it and then call that from the pipeline so we are not repeating the code?

Reply:
@balpurewal For now it's fine here as we already have linting standards in taskctl for wider usage These pre-commit checks are just for data which include Python code etc